lib/permutations.rb

```diff
@@ -1,4 +1,25 @@
-
 def permutations?(string1, string2)
```

Comment:
This doesn't take into account the number of each letter in the strings, so "too" and "tot" would be considered permutations.  I've added a test for C13 to catch this scenario.